common/src/main/java/io/netty/util/HashedWheelTimer.java

```diff
@@ -115,16 +115,16 @@
 
     /**
      * Creates a new timer with the default thread factory
-     * ({@link Executors#defaultThreadFactory()}), default tick duration, and
```

Comment:
@seedeed I think we can't change this in a bug fix release as it will change default behaviour. Some people may depend on it...

Reply:
Hmm... It's a problem. @normanmaurer Without replacing this default thread factory directly, any idea to modify the name of thread created only from Executors.defaultThreadFactory() ?

Reply:
not really ... So I guess we just need to keep it as it is for now.

Reply:
@seedeed use the constructor that takes the thread factory in your application.

Reply:
understood.